### DISCUSSION TITLE
Webpack is hiding logging from a plugin

### DISCUSSION THREAD

**Discussion by solowt** [webpack/webpack#17271] [Q&A] [answered]
Output looks like this:

```
LOG from MergeJsonPlugin
<i> File written to: ./assets/t9n/en.json
<i> File written to: ./assets/t9n/fr.json
<i> File written to: ./assets/t9n/de.json
+ 680 hidden lines
```

I need to see those extra lines but can't figure out how. I tried setting the logger to verbose but it's still hiding all those lines.

**Comment by alexander-akait** [comment]
What do you want to see

**Reply by solowt** [reply]
Whatever those extra 680 lines are

**Reply by solowt** [reply]
I solved the issue I was dealing with but I'm still interested to know how to set the logging for plugins to different levels

**Reply by alexander-akait** [reply]
Do you looking for https://webpack.js.org/configuration/stats/#statslogging?

**Reply by solowt** [reply]
I tried setting that to `verbose` and it printed a lot more stuff but the `+ 680 hidden lines` was still there

**Reply by alexander-akait** [reply]
Can you provide an example?

**Reply by alexander-akait** [reply]
But `hidden lines` from stats printer, so options for stats should enable it

**Reply by solowt** [reply, marked as answer]
I got it to work like by adding this to my config:

```
{
  stats: {
    logging: "verbose",
    loggingDebug: [
      (name) => name === "MyPlugin"
    ]
  }
}
 ```

I didn't include `loggingDebug` earlier.

**Reply by alexander-akait** [reply]
Great :+1:
